swagger.yaml

```diff
@@ -113,6 +113,13 @@ paths:
           type: string
           format: uuid
           required: false
+        - in: query
+          name: ams_subscription_ids
```

Comment:
maybe we should make ``openshift_cluster_id`` plural as well?
@avishayt

Reply:
Yes, but that would break the API.  We can do it as long as it won't break users.

Reply:
Can you open a different task for it @osherdp ?

Reply:
Added some unit-tests and subsystem-tests and extended the manual system-tests described above.

Reply:
https://issues.redhat.com/browse/MGMT-6454

Reply:
Thanks.